src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -22,8 +22,12 @@ CARMACloudPlugin::CARMACloudPlugin(string name) :PluginClient(name) {
         std::lock_guard<mutex> lock(_cfgLock);
 		GetConfigValue<string>("WebServiceIP",webip);

```

Comment:
I don't think GetConfig calls are necessary. I believe Plugin will do this internally automatically with updateConfiguration()

Reply:
I think the initialization requires it, and it follows the same coding paradigm as the other plugins. I agree that the updateConfig is call immediately.  It would be a longer topic if we need to do it for all plugins.